src/Components/Aspire.Azure.AppConfiguration/AzureAppConfigurationSettings.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Azure.Common;
+using Azure.Core;
+
+namespace Aspire.Azure.AppConfiguration;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to Azure App Configuration.
+/// </summary>
+public sealed class AzureAppConfigurationSettings : IConnectionStringSettings
+{
+    /// <summary>
+    /// A <see cref="Uri"/> to the App Config store on which the client operates. Appears as "Endpoint" in the Azure portal.
```

Comment:
Please use either "Azure App Configuration" or "App Configuration" for all occurrences of "App Config".